### PR TITLE
Add commented blank line to fix `Driver` deprecation notice

### LINKED ISSUE #502: `Driver` type does not show as Deprecated in API docs
`NewDriver` shows as deprecated, while `Driver` does not. 

Screenshot from 2023-06-13 04-07-37

### PR DESCRIPTION
**Pull request by fbiville**
Fixes #502

### CODE REVIEW

**Review by bigmontz — APPROVED** [review]
📝

**Review by stefano-ottolenghi — APPROVED** [review]

File: neo4j/resultsummary.go [hunk 1]
```diff
@@ -218,8 +218,9 @@ type Notification interface {
 	// Not all notifications have a unique position to point to and in that case the position would be set to nil.
 	Position() InputPosition
 	// Severity returns the severity level of this notification.
-	// Deprecated: Severity will be removed in 6.0.
 	// Please rely on SeverityLevel (or RawSeverityLevel) instead.
+	//
+	// Deprecated: Severity will be removed in 6.0.
```
Comment (robsdedude) [inline neo4j/resultsummary.go]: That's the wrong way around, isn't it? Shouldn't the "pleas use XYZ" be part of the deprecation message?

Comment (fbiville) [reply neo4j/resultsummary.go]: fixed in f6df686a5c69a0375f26b2521d7ff2e18bae67b2

File: neo4j/result.go [hunk 2]
```diff
@@ -23,8 +23,9 @@ import (
 	"context"
 )
 
+// Result is created via the context-aware Session type.
```
Comment (robsdedude) [inline neo4j/result.go]: Is it? I thought, at least as it stands right now, it's created via either a session (with or without context) or a transaction (with or without context).

**Review by robsdedude — APPROVED** [review]

File: neo4j/result.go (hunk 2 above)
Comment (fbiville) [reply neo4j/result.go]: yeah, my fix is bogus
`Result` is created via `Session`, which is **not** "context-aware.
`ResultWithContext` is created via `SessionWithContext`, which is "context-aware".

Comment (StephenCathcart) [reply neo4j/result.go]:
```suggestion
// Result is created via the Session type.
```

**Review by StephenCathcart — CHANGES_REQUESTED** [review]
👍

**Pull request merged**